tls/s2n_connection.h

```diff
@@ -115,6 +106,17 @@ struct s2n_connection {
     /* If write fd is broken */
     unsigned write_fd_broken:1;
     
+    unsigned quic_enabled:1;
+
+    /* Track request extensions to ensure correct response extension behavior.
```

Comment:
Any reason to move this block? I know there is no wire change, just wondering if it had to do with struct packing or something similar.

Reply:
Yes, I just wanted to keep all the bitflags together.

It doesn't look like S2N focuses on struct packing, but it does look like we try to bundle all our bitflags together, and the extension tracking vars were messing that up.